modules/backing-up-red-hat-quay-standalone.adoc

```diff
@@ -0,0 +1,137 @@
+:_content-type: PROCEDURE
+[[backing-up-red-hat-quay-standalone]]
+= Backing up {productname} on standalone deployments 
+
+This procedure describes how to create a backup of {productname} on standalone deployments. 
+
+.Prerequisites
+
+.Procedure 
+
+. Create a temporary backup directory, for example, `quay-backup`: 
++
+[source,terminal]
+----
+$ mkdir /tmp/quay-backup
+----
+
+. The following example command denotes the local directory that the {productname} was started in, for example, `/opt/quay-install`: 
++
+[source,terminal]
+----
+$ podman run --name quay-app \
+   -v /opt/quay-install/config:/conf/stack:Z \
+   -v /opt/quay-install/storage:/datastorage:Z \
+   {productrepo}/{quayimage}:{productminv}
+----
++
+Change into the directory that bind-mounts to `/conf/stack` inside of the container, for example, `/opt/quay-install`, by running the following command: 
++
+[source,terminal]
+----
+$ cd /opt/quay-install
+----
+
+. Compress the contents of your {productname} deployment into an archive in the `quay-backup` directory by entering the following command: 
++
+[source,terminal]
+----
+$ tar cvf /tmp/quay-backup/quay-backup.tar.gz *
+----
++
+Example output:
++
+[source,terminal]
+----
+config.yaml
+config.yaml.bak
+extra_ca_certs/
+extra_ca_certs/ca.crt
+ssl.cert
+ssl.key
+----
+
+. Back up the Quay container service by entering the following command: 
++
+[source,terminal]
+----
+$ podman inspect quay-app | jq -r '.[0].Config.CreateCommand | .[]' | paste -s -d ' ' -
+
+  /usr/bin/podman run --name quay-app \
+  -v /opt/quay-install/config:/conf/stack:Z \
+  -v /opt/quay-install/storage:/datastorage:Z \
+  {productrepo}/{quayimage}:{productminv}
+----
+
+. Redirect the contents of your `conf/stack/config.yaml` file to your temporary `quay-config.yaml` file by entering the following command: 
++
+[source,terminal]
+----
+$ podman exec -it quay cat /conf/stack/config.yaml > /tmp/quay-backup/quay-config.yaml 
+----
+
+. Obtain the `DB_URI` located in your temporary `quay-config.yaml` by entering the following command:
++
+[source,terminal]
+----
+$ grep DB_URI /tmp/quay-backup/quay-config.yaml
+----
++
+Example output:
++
+----
+$ postgresql://<username>:test123@172.24.10.50/quay
+----
+
+. Extract the PostgreSQL contents to your temporary backup directory in a backup .sql file by entering the following command:
++
+[source,terminal]
+----
+$ pg_dump -h <postgresql_host> -p 5432 -d quay  -U  <username>   -W -O > /tmp/quay-backup/quay-backup.sql
+----
+
+. Print the contents of your `DISTRIBUTED_STORAGE_CONFIG` by entering the following command: 
++
+[source,yaml]
+----
+DISTRIBUTED_STORAGE_CONFIG:
+   default:
+    - S3Storage
+    - s3_bucket: <user_name>          <---------this line
+      storage_path: /registry 
+      s3_access_key: <s3_access_key>
+      s3_secret_key: <s3_secret_key>
+      host: <host_name>
+----
```

Comment:
we can remove ` <---------this line` here. 

`<user_name>` should be `<bucket_name>`.